examples/client/Locomotion/src/pages/ActiveRide/RideDrawer/AddressSelector/AddressLine.js

```diff
@@ -76,19 +76,30 @@ const AddressRow = ({
   border = true,
   actionButton = false,
   onPress,
+  isLoading
 }) => {
   const finalIcon = ICONS[icon];
   return (
-    <Row
-      border
-      onPress={onPress}
-    >
+    <Row border onPress={onPress}>
       <IconContainer>
         {finalIcon ? <Icon Svg={finalIcon} actionButton={actionButton} /> : null}
       </IconContainer>
       <AddressContainer>
+      {isLoading ? (
+        <SkeletonContent
+        containerStyle={{}}
+        isLoading
+        layout={[
+          { width: 180, height: 20, marginBottom: 6 },
+          { width: 220, height: 20},
+        ]}
```

Comment:
what this numbers represent?

Reply:
width and height, im don't think that there is phones with less than 220px